### PR TITLE
feat: implement auto for handling releases

### PR DESCRIPTION
**Pull request by tinytim84**

### CODE REVIEW

File: .github/labels.yml [hunk 1]
```diff
@@ -29,30 +29,33 @@
...
-  description: Increment the patch version when merged
-  color: 067314
-- name: release
-  description: Create a release when this pr is merged
-  color: 38e0d5
-- name: internal
-  description: Changes only affect the internal API
-  color: 696969
-- name: documentation
+- name: 'bump: feature'
```
Comment (AndrewLeedham) [inline .github/labels.yml]: > Maybe there is a better prefix that encompasses both cases? - me

I meant for the chore ones and these.
```suggestion
- name: 'change: feature'
```

**Review by AndrewLeedham — CHANGES_REQUESTED** [review]

File: .github/labels.yml (hunk 1 above)
Comment (tinytim84) [reply .github/labels.yml]: Sorry, I misunderstood.  I've changed them all to `change`.  I've made the label colour different for the two that don't make a release.

File: .github/labels.yml [hunk 2]
```diff
@@ -29,28 +29,28 @@
...
-- name: 'bump: performance'
+- name: 'change: performance'
   description: Improve performance of an existing feature
   color: fcf3b8
-- name: 'bump: dependencies'
+- name: 'change: dependencies'
   description: Updates to dependancies only
   color: fcf3b8
-- name: 'change: test'
+- name: 'change: tests'
```
Comment (AndrewLeedham) [inline .github/labels.yml:53]: The `package.json` version is not pluralised?
https://github.com/xeroxinteractive/config/pull/460/files/e141540a9f3a123fa3af801490ce7bff248b76a8..17daddaaf9d4677d2a67c1761b9a277bce1d1ff8#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R116

**Review by AndrewLeedham — CHANGES_REQUESTED** [review]

File: .github/labels.yml (hunk 2 above)
Comment (tinytim84) [reply .github/labels.yml:53]: Changed.

**Review by AndrewLeedham — APPROVED** [review]

**Pull request merged**